src/Suite.ts

```diff
@@ -15,14 +15,19 @@ async function writeTests() {
   const page = await TestBrowser.createPage(testBrowser, options);
   const stories = await TestBrowser.getStories(page);
   const storiesByComponent = groupBy(stories, 'componentTitle');
+  const patternRegex = new RegExp(options.pattern);
```

Comment:
`.test` requires a RegExp type

Reply:
👍 

We could instantiate the RegExp object in the options parsing, if we wanted. Probably doesn't matter too much one way or the other.

Reply:
ooh right, I like that